askama_shared/src/filters/mod.rs

```diff
@@ -138,6 +139,25 @@ pub fn truncate(s: &fmt::Display, len: &usize) -> Result<String> {
     }
 }
 
+/// Indent lines with `width` spaces
+pub fn indent(s: &fmt::Display, width: &usize) -> Result<String> {
+    let s = format!("{}", s);
```

Comment:
I used `format!` here, to match the existing functions, but I think that `s.to_string()` would be a little more readable.

Reply:
Do you want to change it for all the similar filter functions in one go?

Reply:
Yup, will do.

Reply:
Done